source/command-logging-and-monitoring/command-logging-and-monitoring.rst

```diff
@@ -560,3 +570,4 @@ Changelog
              default port 27017 when relevant. Updated suggested unstructured forms of log messages to more
              clearly label connection IDs and use more readable server address representations.
 :2023-03-23: Updated ``serverConnectionId`` field to be Int64 as long-running servers can return Int64.
+:2023-06-13: Added ``databaseName`` field to ``CommandFailedEvent`` and ``CommandSucceededEvent``.
```

Comment:
This change overlooks two relevant comments from [DRIVERS-2452](https://jira.mongodb.org/browse/DRIVERS-2452):

> Note that to test this we probably need to update the unified test format to add databaseName expectations to expectedCommandEvent.commandSucceededEvent and expectedCommandEvent.commandFailedEvent.

And:

> IIRC, the database name is readily accessible for a CommandStartedEvent because the driver also has access to the outgoing command. Depending how drivers have implemented APM, that may not be the case when emitting CommandFailedEvent and/or CommandSucceededEvent. That should be researched before casually modifying the spec and unified test format.